integration/build.gradle

```diff
@@ -18,6 +18,9 @@ dependencies {
     implementation 'org.junit.jupiter:junit-jupiter-api'
     implementation 'org.hamcrest:hamcrest'
 
+    implementation 'org.glassfish.jersey.core:jersey-client'
```

Comment:
Is this dependency required? I would expect that `jersey-jdk-connector` would be sufficient by itself.